stdlib/Markdown/src/parse/parse.jl

```diff
@@ -8,6 +8,13 @@ mutable struct MD
         new(content, meta)
 end
 
+# For direct messages to the terminal
+struct Message
```

Comment:
Could we (internally) document here that this is only used by the REPL code and the Markdown parser will never put this into the AST? Then we don't have to update Documenter and other users of the Markdown module to support this node type.

Arguably, the `Message` and the `trimdocs` code could all live in the `REPL` module. Or do you think we should make use of this in Documenter as well?